collector/mdadm_linux.go

```diff
@@ -99,6 +99,24 @@ var (
 		[]string{"device"},
 		nil,
 	)
+	blocksSyncedPctDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "md", "blocks_synced_pct"),
+		"Percentage of blocks synced on device.",
+		[]string{"device"},
+		nil,
+	)
+	syncTimeRemainingDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "md", "sync_time_remaining"),
+		"Estimated finishing time for current sync in seconds.",
+		[]string{"device"},
+		nil,
+	)
+	blockSyncedSpeedDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "md", "blocks_synced_speed"),
+		"current sync speed (in Kilobytes/sec)",
+		[]string{"device"},
+		nil,
+	)
```

Comment:
This doesn't seem necessary, we should be able to compute this from something like `rate(node_md_blocks_synced[1m]) * <blocksize>`.

```suggestion
```

Reply:
I think it is usefull. Is is the current speed, as it is shown in /stat/proc/mdstat
I have it showing in my Grafana board.
Plus i guess <blocksize> is not included in the data, so it would require additional configuration for each md-device.

Reply:
My Grafana Board: https://grafana.com/grafana/dashboards/20989-node-exporter-mdadm-status/

Reply:
Note that the groundwork has already been laid for #1085, and we probably should not add any new parsing functionality relating to `/proc/mdstat`.